platform/root/__tests__/index.test.ts

```diff
@@ -98,3 +98,30 @@ test('calls onShutdown param on "shutdown"', async () => {
   expect(onShutdown).toHaveBeenCalledTimes(1);
   expect(onShutdown).toHaveBeenLastCalledWith(err);
 });
+
+test('calls shutdown when configuring logger fails', async () => {
+  const logged = jest.spyOn(console, 'error');
+  logged.mockImplementation(noop);
+
+  const onShutdown = jest.fn();
+
+  const root = new Root(config$, env, onShutdown);
+  const err = new Error('foo bar baz');
+
+  configService.atPath.mockImplementationOnce(() => {
+    throw err;
+  });
+
+  mockServer.mockClear();
+
+  await root.start();
+
+  expect(mockServer).not.toHaveBeenCalled();
+
+  expect(onShutdown).toHaveBeenCalledTimes(1);
+  expect(onShutdown).toHaveBeenLastCalledWith(err);
+
+  expect(logged.mock.calls).toMatchSnapshot();
+
+  logged.mockRestore();
```

Comment:
Not a fan of adding a `beforeEach` + `afterEach` for all tests if it's only used by one test. Would have to be a separate `describe` if so. I can add it.